be/src/common/config.h

```diff
@@ -516,10 +514,23 @@ CONF_Int32(flush_thread_num_per_store, "2");
 // config for tablet meta checkpoint
 CONF_mInt32(tablet_meta_checkpoint_min_new_rowsets_num, "10");
 CONF_mInt32(tablet_meta_checkpoint_min_interval_secs, "600");
+// Thread count of RocksDB uses for background flush and compaction, -1 means the number of cores.
+CONF_mInt32(rocksdb_thread_count, "-1");
+CONF_mInt32(rocksdb_block_cache_mb, "1024");
```

Comment:
Is 1024MB the default value of rocksdb cache? Maybe it is too big?
BTW, `rocksdb_thread_count` and `rocksdb_block_cache_mb` are not mutable configuration, use `CONF_Int32` instead.

Reply:
The default cache is 8M which is too small.
I've updated code to make the block cache shared by all rocksdb instances, and it's capacity is 64M now.